### DISCUSSION TITLE
Stokes flow Information

### DISCUSSION THREAD

**Discussion by JahanzebTariq** [PMEAL/OpenPNM#2652] [Q&A]
![image](https://user-images.githubusercontent.com/99289270/208699137-eb538222-50ee-4334-ba7c-35163f24696e.png)

@jgostick and @ma-sadeghi.. Dear professors I am getting a **0 value of rate** while I saw in the **examples** that it has **some value**.. could you please tell me **what this tells** and is **this value** that I am getting is **correct or not**...?

**Comment by jgostick** [comment]
Hi @JahanzebTariq, are referring to the "0 / 55890" as a the rate being 0? This tells you the number of pores in which you have applied a rate boundary condition.  The rate is found using the ``alg.rate()`` method.

**Reply by JahanzebTariq** [reply]
@jgostick sir **please guide a little more about this rate boundary** condition please.. are my **conditions correct and does this rate stands for the velocity boundary condition.**. and what about the **value boundary condition**..

**Reply by jgostick** [reply]
Here is a detailed tutorial on the process of how openpnm solves the system of equations: https://openpnm.org/examples/reference/simulations/basic_transport.html

Here is an example of find the rate of transport across the domain: https://openpnm.org/examples/applications/absolute_permeability.html

**Reply by jgostick** [reply]
and here is an example that applies both rate and value boundary conditions at the same time (to different pores):
https://openpnm.org/examples/tutorials/08_simulating_transport.html
